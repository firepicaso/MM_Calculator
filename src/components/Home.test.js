import { render } from '@testing-library/react';
import Home from './home';

test('testing Home component', () => {
  const { container } = render(<Home />);
  expect(container.firstChild).toMatchSnapshot();
});
