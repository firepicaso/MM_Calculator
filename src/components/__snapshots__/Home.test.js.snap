// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Home component 1`] = `
<div
  class="homepage"
>
  <h2>
    Welcome to our page!
  </h2>
  <div
    class="welcome"
  >
    <p>
      Welcome to our innovative calculator app that is about sparking your curiosity
    </p>
    <p>
      Our calculator is your reliable companion on your mathematical journey.
    </p>
    <p>
      You will find that math is an adventure waiting to be explored.
    </p>
  </div>
</div>
`;
